CHANGELOG.md

```diff
@@ -12,6 +12,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 ### Changed
 
 ### Removed
+- Remove warning for future Django deprecation [PR #339](https://github.com/model-bakers/model_bakery/pull/339)
```

Comment:
I would consider this `Changed`, not `Removed`, since it is not a removed feature/behavior.

Reply:
@timjklein36 done!